### ISSUE TITLE
Error parsing query with quotes

### ISSUE THREAD

**Issue by duffn** [adelsz/pgtyped#33]
I have a query that looks something like this.

```
/* @name FindThingById */
SELECT
    *
FROM
    database.prefix$environment."Thing"
WHERE
    id = :id;
```

`Thing` was created with quotes, so it is actually `Thing` and not `thing`. 

If I attempt to run `pgtyped` with 

```
npx pgtyped -w -c config.json
```

I receive a parsing error.

```
Parsed file:
1 | /* @name FindThingById */
2 | SELECT
3 |     *
4 | FROM
5 >     database.prefix$environment."Thing"
6 | WHERE
7 |     id = :id;
8 |
9 |
Errors:
- (5:32) Parse error: token recognition error at: '"'
- (5:38) Parse error: token recognition error at: '"'
```

I've tried numerous things to escape the `"` but without success. How can I make this query work with quotes?

**Comment by adelsz** [comment]
Hi @duffn,
Thanks for reporting! Double quotes weren't being parsed correctly.
Fixed this in v0.5.2

**Comment by duffn** [comment]
Thank you for the quick turnaround and excellent package! I can confirm that this works as expected now.
